latex/textbook/clp_series.tex

```diff
@@ -2078,7 +2080,7 @@ \subsection{The Ratio Test}\label{sec:RatioTest}
 So when $x \geq 1$ this is negative and so $f(x)$ is a decreasing function. Thus we can apply the alternating series
```

Comment:
it seems to me $x$ is a bit overloaded here.  It was "ok"-ish before b/c it was big X and little x.  But now...

I think the inner thing should be f(y) not f(x).  Just my $0.02.

Reply:
Joel has updated this to introduce a small t to avoid this overloading.